tests/unit/eblob_wrapper.h

```diff
@@ -0,0 +1,140 @@
+#pragma once
+
+#include <boost/filesystem.hpp>
+
+#include <eblob/eblob.hpp>
+
+#include <library/blob.h>
+#include <library/crypto/sha512.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+
+class item_t {
+public:
+
+	item_t(uint64_t key_, const eblob_key &hashed_key_, const std::vector<char> &value_);
+
+	bool operator< (const item_t &rhs) const;
+
+	bool checked = false;
+	bool removed = false;
+	uint64_t key;
+	eblob_key hashed_key;
+
+	std::vector<char> value;
+};
+
+
+class config_wrapper {
+public:
+	config_wrapper() {
+		data_dir_ = mkdtemp(&data_dir_template_.front());
+		data_path_ = data_dir_ + "/data";
+		log_path_ = data_dir_ + "/log";
+		logger_.reset(new ioremap::eblob::eblob_logger(log_path_.c_str(), EBLOB_LOG_DEBUG));
+
+		config_.blob_flags = EBLOB_L2HASH | EBLOB_DISABLE_THREADS | EBLOB_AUTO_INDEXSORT;
+		config_.sync = -2;
+		config_.log = logger_->log();
+		config_.file = static_cast<char*>(malloc(data_path_.size() + 1));
```

Comment:
Here and for chunks_dir:
I think we don't need malloc here, as lifetime is same with member strings holding that data. despite there is char* , it never changes, so we can do simply:
```suggestion
		config_.file = const_cast<char*>(data_path.c_str());
```

Reply:
done